CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.14.0 (2020-12-01)
+
+- Added modern theme styles to the single file attachment component. [#225](https://github.com/blackbaud/skyux-forms/pull/225)
```

Comment:
Does this mean we need to add single file attachment to the component docs on the modern theme site?

Reply:
Correct.